press/article.md

```diff
@@ -0,0 +1,85 @@
+# Introduction
+
+In the recent react project we needed an environment to create, style and test our components without having a server-side backend code. We decided to use storybook that allows us to develop UI components outside your app and allow other people in your team to work on them. 
+
+The most important benefits of using storybook are :
```

Comment:
Rephrase in a less formal manner, eg  We really enjoy using React Storybook becase: